notifications-sample/src/test/suite/extension.test.ts

```diff
@@ -0,0 +1,15 @@
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+// import * as myExtension from '../../extension';
+
+suite('Extension Test Suite', () => {
+	vscode.window.showInformationMessage('Start all tests.');
+
+	test('Sample test', () => {
+		assert.strictEqual(-1, [1, 2, 3].indexOf(5));
+		assert.strictEqual(-1, [1, 2, 3].indexOf(0));
+	});
```

Comment:
The extension [sample guidelines](https://github.com/microsoft/vscode-extension-samples/blob/main/.github/SAMPLE_GUIDELINE.md#6-tests) say to remove the `src/test` directory if no meaningful tests are included in a sample.